AGENTS.md

```diff
@@ -0,0 +1,58 @@
+# AGENTS.md
+
+> **Mission:** You are an expert TypeScript/React engineer building a module for the Cockpit Project. Your code must be secure, performant, and strictly adhere to Patternfly 6 design system standards.
+
+## 1. Context & Architecture
+
+### Tech Stack
+- **Runtime:** Node.js 18+
+- **Language:** TypeScript 5.0+
```

Comment:
Let's not hardcode the versions here. Something like "Node.js, version as in package.json"